File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    a = 0
    b = 0
    if len(s) % 2 == 1:
        a = len(s) // 2 
        answer = s[a]
    else:
        a = len(s) // 2 - 1
        b = len(s) // 2 
        answer = s[a] + s[b]
    return answer

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    let a = 0;
    let b = 0;
    var answer = '';
    if (s.length % 2 === 1) {
        a = parseInt(s.length / 2);
        answer = s[a];
    } else {
        a = parseInt(s.length / 2) - 1;
        b = parseInt(s.length / 2);
        answer = s[a] + s[b]
    }
    return answer;
}